lib/std/zig/tokenizer.zig

```diff
@@ -1687,6 +1689,16 @@ test "tokenizer - string identifier and builtin fns" {
     });
 }
 
+test "tokenizer - string literal with literal tab" {
+    testTokenize(
+        \\"foo	bar"
+        \\\\foo	bar
```

Comment:
The multi line one is good but the single-line string literal allows no ascii control characters in it